### DISCUSSION TITLE
✅ Ajax search, the most important feature that Elementor lacks.

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#21459] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi elementor team!

In 2023 you are working on great features. But I think now is the time to put Advanced Search Widget (AJAX) on your roadmap.
This is an incredibly important feature that Elementor really needs. 

**Some features required for this widget :**

- The possibility of assigning the search to the desired typing post (For example, I want the user search to find only products and not posts)
- Custom loading (spinner) for searching
- It is possible to display normal and sale prices (WooCommerce)
- Pagination or scrolling for the search results section
- Ability to determine the number of results on each page of search results
- Custom icon for search icon
- Open in new tab option for search results
- Custom text for "No results found"
- Ability to customize the focus mode for the search box

💣Please add other items required by users

### Describe the solution you'd like

please Add an Ajax search widget.

🔥 Finally, I request you to please definitely check out the jet search plugin. thanks

**Comment by brendaha** [comment]
I like this! I've been trying to figure a way around this. I hope they'll fix this immediately.

**Comment by ayoubkhan558** [comment]
+1

**Comment by doronwolf** [comment]
Hi @nicholaszein I think we can close this ticket - Ajax search is available since pro 3.23

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro v3.23! 🥳 

You can now use the [new Search widget with Live Results](https://elementor.com/help/search-widget/), with the option to choose your query as well!

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
